tcptracer-bpf.h

```diff
@@ -91,10 +93,23 @@ struct event_mysql_greeting {
     struct ipv4_tuple_t connection;
 };
 
+struct event_http_response_v6 {
+    __u16 status_code;
+    __u32 response_time;
+    struct ipv6_tuple_t connection;
```

Comment:
looks like it's better to move connection to `struct perf_event` / `struct perf_event_v6` . Otherwise we gonna create two structures for every kind of event.

Reply:
@Q69K how about if I replaced the v4/v6 distinction with this:
```
enum connection_types {
    CONNECTION_TYPE_IPv4 = 0,
    CONNECTION_TYPE_IPv6 = 1,
    CONNECTION_TYPE_MAX
};

union un_connections {
    struct ipv4_tuple_t ipv4_connection;
    struct ipv6_tuple_t ipv6_connection;
};

struct connection_wrapper {
    connection_types connection_type;
    un_connections connection;
};

struct event_http_response {
    __u16 status_code;
    __u32 response_time;
    struct connection_wrapper
};

struct event_mysql_greeting {
    __u16 protocol_version;
    struct connection_wrapper
};
```

Reply:
Then the :arrow_up: `event_http_response` and `event_mysql_greeting` structures can be used for ipv4 and ipv6...

Reply:
That's good!

Just it was initially modelled a bit wrong: http has no idea that there are some IPs, so I'd keep onion like 

```
connection_event {
   address tuple
   union {
       http response  {
           status code
           resp time
       }
       mysql greeting  {
           protocol version
      }
}
```

But anyway is good until it leads to unnecessary duplicates

Reply:
Done